source/standards/production-readiness.html.md.erb

```diff
@@ -0,0 +1,48 @@
+---
+title: How to check if your service is production ready
+last_reviewed_on: 2018-09-01
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+Before taking your service into production and introducing it to live users make sure it meets the relevant technical standards set by GDS and your team.
+
+Some of these standards are provided in the GDS Way but your team may also have its own specific standards. For example, your team may require that all its new services are connected to [Logit][] before going to production.
+
+If you do not implement a standard before people start using your service then you could have difficulties operating your service or supporting it when a fault occurs. **For example…**
+
+## Use a checklist
+
+To avoid problems when your service goes into production you should use a checklist to confirm whether your service is ready. Each check should reference a standard that's already in place before you can take any service into production. **For example…**
+
+You can also include communication checks to make sure specific groups or individuals are informed in advance, **such as...**
+
+Save your check list as a template so your team can apply it each time you need to take a new service or subcomponent into production.
+
+### Example checklist
+
+In order to take a service into production, it must be possible to:
+
+* search logs
+* monitor the health of the service
+* view application metrics
+* alert people when the service has problems
+* follow a documented deployment process
+
+You must also:
+
+* have agreement to proceed from [Information Assurance (IA)][]
+* inform PERSON A
+* inform GROUP B
+
+## Further reading
+
+To find out more about making sure your service is production ready, you can read:
```

Comment:
Not sure if we need this, but it would be helpful if there are any other resources people could read.